src/mtev_cluster.c

```diff
@@ -1222,17 +1222,39 @@ rest_update_cluster(mtev_http_rest_closure_t *restc, int n, char **p) {
   int complete = 0, mask = 0, error_code = 500, status;
   xmlDocPtr indoc, doc;
   xmlNodePtr root;
+  mtev_hash_table *headers = NULL;
+  mtev_boolean force_conf_write = mtev_false;
 
   indoc = rest_get_xml_upload(restc, &mask, &complete);
   if(!complete) return mask;
   if(indoc == NULL) FAIL("xml parse error");
   root = xmlDocGetRootElement(indoc);
   if(!root || strcmp((const char *)root->name, "cluster"))
     FAIL("bad root node: not cluster");
+  headers = mtev_http_request_headers_table(mtev_http_session_request(ctx));
+  if (headers) {
+    char *force_conf_str = NULL;
+    char *hash_retr = "x-mtev-conf-sync";
+    if (mtev_hash_retrieve(headers, hash_retr, strlen(hash_retr), (void **)&force_conf_str)) {
+      int val = atoi(force_conf_str);
```

Comment:
This could be writte a bit clearer with the macros:
```
char *force_conf_str = mtev_hash_dict_get(headers, "x-mtev-conf-str");
if(force_conf_str) {
```